docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -86,6 +88,7 @@ Example:
 |--------	|--------------|:-------:|:--------:|
 | `namespace` | Namespace to deploy the Prometheus Operator. | - | true |
 | `grafana_admin_password` | Password for `admin` user in Grafana.  | - | true |
+| `grafana_ingress_host` | Ingress URL Host to expose Grafana over internet. **NOTE:** When running on Packet, a DNS entry pointing at the ingress controller needs to be created.  | - | false |
```

Comment:
- Could you please fix the capitalization?
- There's a missing article before "internet".

Reply:
> Could you please fix the capitalization?

I see that only `Host` has unnecessary capitalisation?

Reply:
Yup, looks good now.